samples/display/cfb/README.rst

```diff
@@ -0,0 +1,35 @@
+.. _character_frame_buffer_sample:
+
+Character frame buffer
+######################
+
+Overview
+********
+
+This sample will display a vertical scrolling text string making use of the
+character frame buffer framework.
+
+Requirements
+************
+
+This sample requires a board and a display, the display needs to be supported
```

Comment:
How about:

    This sample requires a supported board and CFB-supporting
    display, such as the :ref:`reel_board`.